api/envoy/extensions/watchdog/abort_action/v3alpha/abort_action.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package envoy.extensions.watchdog.abort_action.v3alpha;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.watchdog.abort_action.v3alpha";
+option java_outer_classname = "AbortActionProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Watchdog Action that sends a SIGABRT to kill the process.]
+// [#extension: envoy.watchdog.abort_action]
+
+// Configuration for the profile watchdog action.
```

Comment:
Comment seems wrong/out of date. Note that this comment is the primary documentation for this in the generated docs. It should include some information about what this does, when/why use it, etc.

Reply:
Done. Added additional information from the action's c++ implementation as you suggested.